test/TestProjects/MgmtResourceName/readme.md

```diff
@@ -0,0 +1,18 @@
+# ExactMatchInheritance
+
+## AutoRest Configuration
+
+> see https://aka.ms/autorest
+
+``` yaml
+title: MgmtResourceName
+require: $(this-folder)/../../../readme.md
+azure-arm: true
+model-namespace: false
+input-file: $(this-folder)/MgmtResourceName.json
+namespace: MgmtResourceName
+
+
+request-path-to-resource-name:
```

Comment:
Add a test case that does a client rename and verify in test that it still adds Resource

Reply:
added a test case for `rename-model`